src/khoj/configure.py

```diff
@@ -67,7 +67,7 @@ async def authenticate(self, request):
             user = await self.khojuser_manager.filter(email=current_user.get("email")).afirst()
             if user:
```

Comment:
In the `authenticate` method of the middleware, we should also check whether the `current_user`'s session object has an `exp` timestamp that exceeds the current time.